app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsBackupController.kt

```diff
@@ -287,12 +401,20 @@ class SettingsBackupController : SettingsController() {
 
         private companion object {
             const val KEY_URI = "RestoreBackupDialog.uri"
+            const val KEY_TYPE = "RestoreBackupDialog.type"
+            const val KEY_MODE = "RestoreBackupDialog.mode"
         }
     }
 
     private companion object {
         const val CODE_BACKUP_CREATE = 501
         const val CODE_BACKUP_RESTORE = 502
         const val CODE_BACKUP_DIR = 503
+        const val CODE_FULL_BACKUP_CREATE = 504
+        const val CODE_FULL_BACKUP_RESTORE = 505
+        const val TYPE_LITE = 1
+        const val TYPE_FULL = 2
+        const val MODE_ONLINE = true
+        const val MODE_OFFLINE = false
```

Comment:
It's not very clear what the difference between type and mode is when just looking at this.